CONTRIBUTING.md

```diff
@@ -9,17 +9,21 @@
 0. Start postgres if it isn't automatically running: `postgres -D /usr/local/var/postgres`
 0. Install Elasticsearch and make sure it is running.
 0. Install Redis and make sure it is running.
+0. Install the [Qt implementation of Webkit](https://github.com/thoughtbot/capybara-webkit/wiki/Installing-Qt-and-compiling-capybara-webkit) as we now depend on [capybara-webkit gem](https://github.com/thoughtbot/capybara-webkit).
 0. Run `bundle install` to grab the required gems.
 0. Use `rake db:create:all` to create the databases.
 0. User `rake db:migrate` to transform the database structure to what is
    needed by the app. For test you will need to preface with env variables 
    to indicate the environment: `RACK_ENV=test rake db:migrate`
-0. Place environment-specific variables in a `.env.`*environmentname* file. The default is `.env`.
+0. Run `./bin/setup` to initialize the rails application.
+0. Edit the environment-specific variables in a `.env.`*environmentname* file. The default created by running `./bin/setup` is `.env`.
    Test-specific variables can go in `.env.test`. If you have set a postgres username and password,
    use the `DATABASE_URL` environment variable to configure it. See `.env.example` for an example.
    Default database configuration is stored in `config/database.yml`.
-0. Run `rake serve`. This will build the Jekyll site and start the Sinatra server.
-0. Go to `localhost:9292` and enjoy!
+0. Add the CROMERR account credentials (CDX_USERNAME and CDX_PASSWORD) to `.env`.
+0. Run `rake secret` and set the output as the value for `SECRET_KEY_BASE` in `.env`.
+0. Run `rails s` to start the Rails application.
+0. Go to `http://localhost:3000` and enjoy!
```

Comment:
foreman does not default to `localhost:3000` like the standard rails server. Perhaps we should add a foreman config for PORT so we can tell ppl where to do. Otherwise, seems like it assigns randomly (mine is at 5100) https://robots.thoughtbot.com/foreman-as-process-manager